src/app/sections/ProbeDetails.svelte

```diff
@@ -122,26 +170,53 @@ h2 {
     {/if}
     <div class=drawer-section>
         {#if $store.probe.description}
-            <h2 class=detail__heading--01>description</h2>
-            <div class='probe__description helper-text--01'>
+            <h2 class="detail__heading--01">description</h2>
+            <div class="probe-description helper-text--01">
                 {@html $store.probe.description}
             </div>
         {/if}
     </div>
     {#if $store.probe.bugs && $store.probe.bugs.length}
     <div class="drawer-section">
-            <h2 class=detail__heading--01>associated bugs</h2>
-            <div class="bug-list helper-text--01">
-            {#each $store.probe.bugs as bugID, i (bugID)}
-                <a
-                href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
-                target="_blank">{bugID}</a>
-            {/each}
-            </div>
+        <h2 class="detail__heading--01">associated bugs</h2>
+        <div class="bug-list helper-text--01">
+        {#each $store.probe.bugs as bugID, i (bugID)}
+            <a
+            href='https://bugzilla.mozilla.org/show_bug.cgi?id={bugID}'
+            target="_blank">{bugID}</a>
+        {/each}
+        </div>
     </div>
     {/if}
-    <div class='drawer-section drawer-section--end'>
-        <h2 class=detail__heading--01>export</h2>
+    <div class="drawer-section">
+        <h2 class="detail__heading--01">extras</h2>
+        <dl class="details-list">
+        {#if $store.probe.active !== undefined}
+            <dt><span class="label label-text--01 label--definition-title">active</span></dt>
+            <dd class="details-list-definition label-text--01">{$store.probe.active}</dd>
+        {/if}
+        {#if $store.probe.prerelease !== undefined}
+            <dt><span class="label label-text--01 label--definition-title">pre-release</span></dt>
+            <dd class="details-list-definition label-text--01">{$store.probe.prerelease}</dd>
+        {/if}
+        </dl>
+    </div>
+    {#if $store.versions && $store.versions.length}
+    <div class="drawer-section">
+        <header class="drawer-header drawer-header--split">
+            <h2 class="detail__heading--01">available in versions</h2>
+            <span class="label-text--01">{$store.channel}</span>
+        </header>
+        <dl class="details-list">
+            <dt><span class="label label-text--01 label--definition-title">first</span></dt>
```

Comment:
similarly, we might just say `beta     47 - 70` here instead of having the first, last, and beta info as separate entities